TiledLib.Tests/Data/External_tileset.tsx

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is just a re-saved version of external_tileset.json.
The json file is unchanged except I added a wang set and filled in a few tiles for it.